src/main/resources/solr-home/conf/schema.xml

```diff
@@ -85,64 +85,71 @@
-        <field name="altidB"			type="miqlDummyField" default="-"/>
-        <field name="aliasA"			type="miqlDummyField" default="-"/>
-        <field name="aliasB"			type="miqlDummyField" default="-"/>
+        <field name="altidA"		type="miqlDummyField" default="-"/>
+        <field name="altidB"		type="miqlDummyField" default="-"/>
+        <field name="aliasA"		type="miqlDummyField" default="-"/>
+        <field name="aliasB"		type="miqlDummyField" default="-"/>
 
-        <!-- MIQL 2.7 indexed and stored fields http://code.google.com/p/psicquic/wiki/MiqlReference27 -->
+        <!-- MIQL 2.7 indexed and stored fields http://psicquic.github.io/MiqlReference27.html -->
         <field name="complex" 		type="miqlText" 	default="-"/>
-        <field name="pbioroleA" 		type="miqlText" 	default="-"/>
-        <field name="pbioroleB" 		type="miqlText"		default="-"/>
+        <field name="pbioroleA" 	type="miqlText" 	default="-"/>
+        <field name="pbioroleB" 	type="miqlText"		default="-"/>
         <field name="pbiorole" 		type="miqlText" 	default=""/> <!-- pbiorole = pbioroleA + pbioroleB -->
         <field name="ptypeA" 		type="miqlText" 	default="-"/>
         <field name="ptypeB" 		type="miqlText" 	default="-"/>
-        <field name="ptype" 			type="miqlText" 	default=""/> <!-- ptype = ptypeA + ptypeB -->
+        <field name="ptype" 		type="miqlText" 	default=""/> <!-- ptype = ptypeA + ptypeB -->
         <field name="pxrefA" 		type="miqlText" 	default="-"/>
         <field name="pxrefB" 		type="miqlText" 	default="-"/>
-        <field name="pxref" 			type="miqlText" 	default=""/> <!-- pxref = pxrefA + pxrefB -->
+        <field name="pxref" 		type="miqlText" 	default=""/> <!-- pxref = pxrefA + pxrefB -->
         <field name="xref" 			type="miqlText" 	default="-"/>
-        <field name="annot" 			type="miqlText" 	default=""/>
-        <field name="udate" 			type="miqlInt"  	default="0"/>
+        <field name="annot" 		type="miqlText" 	default=""/>
+        <field name="udate" 		type="miqlInt"  	default="0"/>
         <field name="negative" 		type="miqlBoolean" 	default="false"/>
         <field name="ftypeA" 		type="miqlText" 	default="-"/>
         <field name="ftypeB" 		type="miqlText" 	default="-"/>
-        <field name="ftype" 			type="miqlText" 	default=""/> <!-- ftype = ftypeA + ftypeB -->
+        <field name="ftype" 		type="miqlText" 	default=""/> <!-- ftype = ftypeA + ftypeB -->
         <field name="pmethodA"		type="miqlText" 	default="-"/>
         <field name="pmethodB" 		type="miqlText" 	default="-"/>
         <field name="pmethod" 		type="miqlText" 	default=""/> <!-- pmethod = pmethodA + pmethodB -->
         <field name="stc" 			type="miqlBoolean"  default="false"/>
-        <field name="param" 			type="miqlBoolean"  default="false"/>
+        <field name="param" 		type="miqlBoolean"  default="false"/>
+
+        <!-- MIQL 2.8 indexed and stored fields !-->
```

Comment:
TODO: Add the reference to the extension of MIQL when is available in the PSICQUIC documentation

Reply:
It's already done :)